nautobot/dcim/views.py

```diff
@@ -1189,6 +1279,30 @@ class DeviceBayTemplateBulkDeleteView(generic.BulkDeleteView):
     filterset = filters.DeviceBayTemplateFilterSet
 
 
+#
+# Module bay templates
+#
+
+
+class ModuleBayTemplateUIViewSet(
+    ObjectEditViewMixin,
+    ObjectDestroyViewMixin,
+    ObjectBulkDestroyViewMixin,
+    ObjectBulkUpdateViewMixin,
+):
+    queryset = ModuleBayTemplate.objects.all()
+    bulk_update_form_class = forms.ModuleBayBulkEditForm
+    filterset_class = filters.ModuleBayTemplateFilterSet
+    # filterset_form_class = forms.ModuleBayTemplateFilterForm
+    form_class = forms.ModuleBayTemplateForm
+    serializer_class = serializers.ModuleBayTemplateSerializer
+    # table_class = tables.ModuleBayTemplateTable
+
+    @action(detail=False, url_path="rename", url_name="bulk_rename")
+    def add(self, request, *args, **kwargs):
+        return None
```

Comment:
TODO I presume?

Reply:
Done now but there are still a few custom actions I need to implement for these models.